Cleanup: Audit log and error capitalization

Generally: log-lines start with capital letters; error strings do not (unless it starts with an acronym like IP).  We're not consistent with this.  It should be relatively easy to audit these and fix-up.

Ooh, @krousey notes that `golint` already does this, but you have to lower the `min_confidence`. Then you get errors like this: `runners.go:77:25: error strings should not be capitalized and should not end with punctuation`

--- a/cluster/images/etcd-version-monitor/etcd-version-monitor.go
+++ b/cluster/images/etcd-version-monitor/etcd-version-monitor.go
@@ -19,6 +19,7 @@ package main
 import (
 	"bytes"
 	"encoding/json"
+	"errors"
 	goflag "flag"
 	"fmt"
 	"net/http"
@@ -202,21 +203,21 @@ func getVersion(lastSeenBinaryVersion *string) error {
 	// Create the get request for the etcd version endpoint.
 	req, err := http.NewRequest("GET", etcdVersionScrapeURI, nil)
 	if err != nil {
-		return fmt.Errorf("Failed to create GET request for etcd version: %v", err)
+		return fmt.Errorf("failed to create GET request for etcd version: %v", err)
 	}
 
 	// Send the get request and receive a response.
 	client := &http.Client{}
 	resp, err := client.Do(req)
 	if err != nil {
-		return fmt.Errorf("Failed to receive GET response for etcd version: %v", err)
+		return fmt.Errorf("failed to receive GET response for etcd version: %v", err)
 	}
 	defer resp.Body.Close()
 
 	// Obtain EtcdVersion from the JSON response.
 	var version EtcdVersion
 	if err := json.NewDecoder(resp.Body).Decode(&version); err != nil {
-		return fmt.Errorf("Failed to decode etcd version JSON: %v", err)
+		return fmt.Errorf("failed to decode etcd version JSON: %v", err)
 	}
 
 	// Return without updating the version if it stayed the same since last time.
@@ -228,7 +229,7 @@ func getVersion(lastSeenBinaryVersion *string) error {
 	if *lastSeenBinaryVersion != "" {
 		deleted := etcdVersion.Delete(prometheus.Labels{"binary_version": *lastSeenBinaryVersion})
 		if !deleted {
-			return fmt.Errorf("Failed to delete previous version's metric")
+			return errors.New("failed to delete previous version's metric")
 		}
 	}
 
@@ -259,14 +260,14 @@ func getVersionPeriodically(stopCh <-chan struct{}) {
 func scrapeMetrics() (map[string]*dto.MetricFamily, error) {
 	req, err := http.NewRequest("GET", etcdMetricsScrapeURI, nil)
 	if err != nil {
-		return nil, fmt.Errorf("Failed to create GET request for etcd metrics: %v", err)
+		return nil, fmt.Errorf("failed to create GET request for etcd metrics: %v", err)
 	}
 
 	// Send the get request and receive a response.
 	client := &http.Client{}
 	resp, err := client.Do(req)
 	if err != nil {
-		return nil, fmt.Errorf("Failed to receive GET response for etcd metrics: %v", err)
+		return nil, fmt.Errorf("failed to receive GET response for etcd metrics: %v", err)
 	}
 	defer resp.Body.Close()
 

